src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsService.java

```diff
@@ -1,27 +1,11 @@
 package it.gov.pagopa.pu.classification.service;
 
-import it.gov.pagopa.pu.classification.connector.debtposition.InstallmentNoPIIService;
+import it.gov.pagopa.pu.classification.model.Assessments;
 import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentNoPIIResponse;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.context.annotation.Lazy;
-import org.springframework.stereotype.Service;
 
 import java.util.List;
 
-@Lazy
-@Slf4j
-@Service
-public class AssessmentsService {
-
-  private final InstallmentNoPIIService installmentNoPIIService;
-
-
-  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService) {
-    this.installmentNoPIIService = installmentNoPIIService;
-  }
-
-  public List<InstallmentNoPIIResponse> getInstallmentsByReceiptId(Long receiptId, String accessToken) {
-    return installmentNoPIIService.getByReceiptId(receiptId, accessToken);
-  }
-
+public interface AssessmentsService {
+  List<InstallmentNoPIIResponse> getInstallmentsByReceiptId(Long receiptId, String accessToken);
```

Comment:
Should this method really be exposed? I don't see the point of such a method for a Service called (AssessmentsService) not as public...